packages/internal/test-util/src/lib/spy-driver/spy-driver-root.module.ts

```diff
@@ -1,38 +1,22 @@
-import { NgModule, Optional, SkipSelf } from '@angular/core';
+import { inject, NgModule, InjectFlags } from '@angular/core';
 
-import {
-  LumberjackLogDriverConfig,
-  lumberjackLogDriverConfigToken,
-  lumberjackLogDriverToken,
-} from '@ngworker/lumberjack';
+import { lumberjackLogDriverToken } from '@ngworker/lumberjack';
 
-import { spyDriverConfigToken } from './spy-driver-config.token';
-import { SpyDriverConfig } from './spy-driver.config';
 import { SpyDriver } from './spy.driver';
 
-export function spyDriverFactory(
-  logDriverConfig: LumberjackLogDriverConfig,
-  spyDriverConfig: SpyDriverConfig
-): SpyDriver {
-  const baseConfig = { ...logDriverConfig, identifier: SpyDriver.driverIdentifier };
-  const fullConfig = { ...baseConfig, ...spyDriverConfig };
-
-  return new SpyDriver(fullConfig);
-}
-
 @NgModule({
   providers: [
     {
       provide: lumberjackLogDriverToken,
-      useFactory: spyDriverFactory,
-      deps: [lumberjackLogDriverConfigToken, spyDriverConfigToken],
+      useClass: SpyDriver,
       multi: true,
     },
   ],
 })
 export class SpyDriverRootModule {
-  constructor(@Optional() @SkipSelf() maybeNgModuleFromParentInjector?: SpyDriverRootModule) {
-    if (maybeNgModuleFromParentInjector) {
+  maybeNgModuleFromParentInjector = inject(SpyDriverRootModule, InjectFlags.Optional | InjectFlags.SkipSelf);
```

Comment:
issue: mark as private.
```suggestion
  private maybeNgModuleFromParentInjector = inject(SpyDriverRootModule, InjectFlags.Optional | InjectFlags.SkipSelf);
```

Reply:
nit: separate property and constructor with a blank line.